linode_api4/objects/networking.py

```diff
@@ -9,20 +9,56 @@ class IPv6Pool(Base):
     properties = {
         "range": Property(identifier=True),
         "region": Property(slug_relationship=Region, filterable=True),
+        "prefix": Property(),
+        "route_target": Property(),
     }
 
+    def _api_get(self):
+        """
+        A helper method to GET this object from the server
+        """
+        pools = self._client.networking.ipv6_pools()
+        pool = [p for p in pools if p.range == self.range][0]
+
+        if pool is not None:
+            self._populate(pool._raw_json)
+        else:
+            raise LookupError("Could not find IPv6 pool with proviced range.")
+
 
 class IPv6Range(Base):
-    api_endpoint = "/networking/ipv6/ranges/{}"
+    api_endpoint = "/networking/ipv6/ranges/{range}"
     id_attribute = "range"
 
     properties = {
         "range": Property(identifier=True),
         "region": Property(slug_relationship=Region, filterable=True),
+        "prefix": Property(),
+        "route_target": Property(),
     }
 
+    def ip_ranges_list(self):
+        """
+        Displays the IPv6 ranges on your Account.
+        """
+
+        result = self._client.post("/networking/ipv6/ranges", model=self)
+
+        return [IPv6Range(self._client, r["range"]) for r in result["data"]]
+
+    def ip_range_delete(self):
```

Comment:
I think we can drop this method since `.delete()` is inherited from the `Base` class 👍